spec/lib/va_profile/v2/contact_information/service_spec.rb

```diff
@@ -28,13 +30,14 @@
         end
       end
 
-      it 'supports international provinces' do
-        VCR.use_cassette('va_profile/v2/contact_information/person_intl_addr', VCR::MATCH_EVERYTHING) do
-          response = subject.get_person
+      # Need an international user
```

Comment:
Is there any reason this is commented out?

Reply:
I will need to get with Tom about the international address.